### ISSUE TITLE
Letsencrypt renewal fails with multiple intermediate certificates on GL <= 4.2.6

### ISSUE THREAD

**Issue by sengelsman** [globaleaks/GlobaLeaks#3006]
When trying to renew the letsencrypt certificate globaleaks crashes.

Since 2021-05-10 00:00:00 the attempted renewal of the certificate on one of our globaleaks instances fails and leaves globaleaks unavailable.

I added some printing of the intermediate certificates at renewal time and noticed the certs where concatenated as follows:
```
-----END CERTIFICATE----------BEGIN CERTIFICATE-----
```

I think this could be easily fixed by patching `util/letsencrypt.py`

```python
    return certificates[0], ''.join(certificates[1:])
```
to 

```python
    return certificates[0], '\n'.join(certificates[1:])
```

```
2021-05-11 00:00:29+0000 [-] [E] Job CertificateCheck died with runtime -1.0000 [low: -1.0000, high: -1.0000]
2021-05-11 00:00:29+0000 [-] Traceback (most recent call last):
2021-05-11 00:00:29+0000 [-]   File "/usr/lib/python3/dist-packages/globaleaks/jobs/job.py", line 49, in run
2021-05-11 00:00:29+0000 [-]     yield self.operation()
2021-05-11 00:00:29+0000 [-]   File "/usr/lib/python3/dist-packages/twisted/internet/defer.py", line 1384, in _inlineCallbacks
2021-05-11 00:00:29+0000 [-]     result = result.throwExceptionIntoGenerator(g)
2021-05-11 00:00:29+0000 [-]   File "/usr/lib/python3/dist-packages/twisted/python/failure.py", line 408, in throwExceptionIntoGenerator
2021-05-11 00:00:29+0000 [-]     return g.throw(self.type, self.value, self.tb)
2021-05-11 00:00:29+0000 [-]   File "/usr/lib/python3/dist-packages/globaleaks/jobs/certificate_check.py", line 88, in operation
2021-05-11 00:00:29+0000 [-]     yield self.cert_expiration_checks(tid)
2021-05-11 00:00:29+0000 [-]   File "/usr/lib/python3/dist-packages/twisted/python/threadpool.py", line 250, in inContext
2021-05-11 00:00:29+0000 [-]     result = inContext.theWork()
2021-05-11 00:00:29+0000 [-]   File "/usr/lib/python3/dist-packages/twisted/python/threadpool.py", line 266, in <lambda>
2021-05-11 00:00:29+0000 [-]     inContext.theWork = lambda: context.call(ctx, func, *args, **kw)
2021-05-11 00:00:29+0000 [-]   File "/usr/lib/python3/dist-packages/twisted/python/context.py", line 122, in callWithContext
2021-05-11 00:00:29+0000 [-]     return self.currentContext().callWithContext(ctx, func, *args, **kw)
2021-05-11 00:00:29+0000 [-]   File "/usr/lib/python3/dist-packages/twisted/python/context.py", line 85, in callWithContext
2021-05-11 00:00:29+0000 [-]     return func(*args,**kw)
2021-05-11 00:00:29+0000 [-]   File "/usr/lib/python3/dist-packages/globaleaks/orm.py", line 138, in _wrap
2021-05-11 00:00:29+0000 [-]     result = function(self.instance, session, *args, **kwargs)
2021-05-11 00:00:29+0000 [-]   File "/usr/lib/python3/dist-packages/globaleaks/jobs/certificate_check.py", line 77, in cert_expiration_checks
2021-05-11 00:00:29+0000 [-]     self.state.snimap.load(tid, tls_config)
2021-05-11 00:00:29+0000 [-]   File "/usr/lib/python3/dist-packages/globaleaks/utils/sni.py", line 144, in load
2021-05-11 00:00:29+0000 [-]     conf['ssl_intermediate'])
2021-05-11 00:00:29+0000 [-]   File "/usr/lib/python3/dist-packages/globaleaks/utils/tls.py", line 195, in __init__
2021-05-11 00:00:29+0000 [-]     x509 = load_certificate(FILETYPE_PEM, intermediate)
2021-05-11 00:00:29+0000 [-]   File "/usr/lib/python3/dist-packages/OpenSSL/crypto.py", line 1824, in load_certificate
2021-05-11 00:00:29+0000 [-]     _raise_current_error()
2021-05-11 00:00:29+0000 [-]   File "/usr/lib/python3/dist-packages/OpenSSL/_util.py", line 54, in exception_from_error_queue
2021-05-11 00:00:29+0000 [-]     raise exception_type(errors)
2021-05-11 00:00:29+0000 [-] OpenSSL.crypto.Error: [('PEM routines', 'get_header_and_data', 'bad end line')]
```
 - OS: ubuntu 18.04.5
 - Globaleaks 4.1.17

**Comment by sengelsman** [comment]
I can't confirm yet since i hit the daily rate limit of letsencrypt trying to debug this. I hope to have an answer tomorrow morning

**Comment by evilaliv3** [comment]
I see! Actually this bug was fixed by me on 4.2.7.

I got confused by you message where you actually were referring 4.1.17!

This is the fix applied: https://github.com/globaleaks/GlobaLeaks/commit/11f808892bc0d2b536d0d04a6292b05212c20053
